### DISCUSSION TITLE
load a float number drops digits

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#371] [Q&A]
I tried the following:
`load r1,3.141592653589793238462643383279      /* Load first float */    say r1`
after loading it the contents is shown as:
` 3.14159`
a double should be able to keep 15 digits. Am I thinking wrong?

**Comment by adesutherland** [comment]
Yeah - please raise it as an issue in the assembler—maybe it is using float instead of double in the conversion :-(
